tests/workloads/fastmri/compare.py

```diff
@@ -0,0 +1,111 @@
+import os
+
+# Disable GPU access for both jax and pytorch.
+os.environ["CUDA_VISIBLE_DEVICES"] = ""
+
+from flax import jax_utils
+import jax
+import numpy as np
+import torch
+
+from algorithmic_efficiency import spec
+from algorithmic_efficiency.workloads.fastmri.fastmri_jax.workload import \
+    FastMRIWorkload as JaxWorkload
+from algorithmic_efficiency.workloads.fastmri.fastmri_pytorch.workload import \
+    FastMRIWorkload as PytWorkload
+from tests import torch2jax_utils as utils
+
+
+def sd_transform(sd):
+
+  def sort_key(k):
+    if k[0] == 'ModuleList_0':
+      return (0, *k)
+    if k[0] == 'ConvBlock_0':
+      return (1, *k)
+    if k[0] == 'ModuleList_1':
+      return (2, *k)
+    if k[0] == "ModuleList_2":
+      return (3, *k)
+
+  keys = sorted(sd.keys(), key=sort_key)
+  c = 0
+  for idx, k in enumerate(keys):
+    new_key = []
+    for idx2, i in enumerate(k):
+      if 'ModuleList' in i or 'Sequential' in i:
+        continue
+      if i.startswith('ConvBlock'):
+        if idx != 0 and keys[idx - 1][:idx2 + 1] != k[:idx2 + 1]:
+          c += 1
+        i = f'ConvBlock_{c}'
+      if 'Conv2d' in i:
+        i = i.replace('Conv2d', 'Conv')
+      if 'ConvTranspose2d' in i:
+        i = i.replace('ConvTranspose2d', 'ConvTranspose')
+      if 'weight' in i:
+        i = i.replace('weight', 'kernel')
+      new_key.append(i)
+    new_key = tuple(new_key)
+    sd[new_key] = sd[k]
+    del sd[k]
+  return sd
+
+
+def value_transform(k, value, jax_value):
```

Comment:
it seems like all of the `value_transform` functions in the `compare.py` files could be combined into a single function, which would be safer to avoid any copied errors. could you merge them all and put that in torch2jax_utils.py?

Reply:
Done!